qml/Crypto/main.cpp

```diff
@@ -74,6 +76,11 @@ Sailfish::Crypto::Key Sailfish::Crypto::Plugin::CryptoManager::constructKey() co
     return Sailfish::Crypto::Key();
 }
 
+Sailfish::Crypto::Key Sailfish::Crypto::Plugin::CryptoManager::constructKeyTemplate(const QString &name, const QString &collectionName, const QString &storagePluginName) const
```

Comment:
This is essentially an overload of `constructKey` ― could you rename it as such?

Reply:
That's reasonable, done.